layouts/shortcodes/translations.html

```diff
@@ -1,5 +1,5 @@
 <ul class="translation-list">
   {{ range $.Site.Home.AllTranslations }}
-    <li><a href="{{ .Permalink }}">{{ .Language.LanguageName }}</a></li>
+    <li><a href="{{ .Permalink }}" lang="{{ .Language.LanguageCode }}" hreflang="{{ .Language.LanguageCode }}">{{ .Language.LanguageName }}</a></li>
```

Comment:
Try this:

```
<ul class="translation-list">
  {{ range $.Site.Home.AllTranslations }}
    <li>
      <a href="{{ .Permalink }}" lang="{{ .Language }}" hreflang="{{ .Language }}">
        {{ .Language.LanguageName }}
      </a>
    </li>
{{ end }}
</ul>
```

Reply:
It works but I found out that the Spanish langcode was not good so I added a new commit to fix it :)